src/System.Security.Cryptography.Encoding/src/System/Security/Cryptography/Base64Transforms.cs

```diff
@@ -345,21 +367,27 @@ internal class ThrowHelper
         public static void ValidateTransformBlock(byte[] inputBuffer, int inputOffset, int inputCount)
         {
             if (inputBuffer == null) ThrowArgumentNull(ExceptionArgument.inputBuffer);
-            if ((uint)inputCount > inputBuffer.Length) ThrowArgumentInvalidValue(ExceptionArgument.inputCount);
-            if (inputOffset < 0) ThrowArgumentOutOfRangeNeedNonNegNum(ExceptionArgument.inputOffset);
+
+            if ((uint)inputCount > inputBuffer.Length) ThrowArgumentOutOfRange(ExceptionArgument.inputCount);
+
+            if (inputOffset < 0) ThrowArgumentOutOfRange(ExceptionArgument.inputOffset);
```

Comment:
Sorry, I meant

```C#
if ((uint)inputCount > inputBuffer.Length)
    ThrowArgumentInvalidValue(ExceptionArgument.inputCount);
if (inputOffset < 0)
    ThrowArgumentOutOfRangeNeedNonNegNum(ExceptionArgument.inputOffset);
```

From rule 1 in https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md:
> A single line statement block can go without braces but the block must be properly indented on its own line